app/presenters/scope_filters_presenter.rb

```diff
@@ -1,13 +1,17 @@
 require 'gds_api/support_api'
 
 class ScopeFiltersPresenter
-  attr_reader :path, :organisation_slug
+  attr_reader :organisation_slug
 
   def initialize(path: nil, organisation_slug: nil)
-    @path = path
+    @requested_path = path
     @organisation_slug = organisation_slug
   end
 
+  def path
+    @path ||= path_from_requested_path if requested_path.present?
```

Comment:
Can `path` be renamed to `parsed_path` so we can tell from the start that it's the result of doing things to the requested path? 

There might be some confusion about what the difference is between `path` and `requested_path`, especially since it shares a name with a parameter sent to the `initialize` method above. And that parameter refers to the original path, before any parsing.

Reply:
Good call, did a0b7e44c1159e63dd0b5e061351cfa8f9fc79e5a as a fixup commit which I'll squash down if you think it's clearer (or I will take another crack at if you don't)